Yoast/wordpress-seo
# Check if there are installed addons first before doing things

## Summary


This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Have installed addons subscriptions
* Verify the premium page still works like before
* Merge this branch in premium and verify the metabox doens't show a yellow warning that your Premium license isn't active.

## UI changes

## Documentation

## Quality assurance


Fixes #
